components/decimal/src/error.rs

```diff
@@ -6,10 +6,11 @@
 
 use displaydoc::Display;
 
-#[allow(missing_docs)] // TODO(#1025) - Add missing docs.
+/// A list of possible error outcomes for the [`FixedDecimal`](crate::FixedDecimal) struct.
```

Comment:
Suggestion: This doc is not correct. `FixedDecimal` is not in this crate.

I think we should probably change our language here to make it more general since these errors are typically intended to cover an entire crate. How about

```suggestion
/// A list of possible error outcomes for operations in this crate.
```

or, if you want to be more specific, at risk of copy/paste errors,

```suggestion
/// A list of possible error outcomes for operations in the [`icu_decimal`](crate) crate.
```

(not sure if that syntax works)